src/main/java/com/client/config/ScheduledEventConfig.java

```diff
@@ -0,0 +1,77 @@
+package com.client.config;
```

Comment:
What if we had a folder called "subscribedevents" (Name does not matter). In each folder, you create your four java beans for the scheduled task that has a subscription associated with it. Then there would be one scheduled task configuration for each task that is set up?

Reply:
Discussed over slack and PR 1 in your fork has a possible solution to this.